modules/installation-deployment-manager-int-lb.adoc

```diff
@@ -15,3 +15,5 @@ You can use the following Deployment Manager template to deploy the internal loa
 include::https://raw.githubusercontent.com/openshift/installer/release-4.8/upi/gcp/02_lb_int.py[]
 ----
 ====
+
+You will need this in addition to the `02_lb_ext.py` when you create external cluster.
```

Comment:
Possibly sprinkle "template" in a few places? Also seems to be missing the word "an" (or maybe "the") before "external cluster". E.g.:

```
You will need this template in addition to the `02_lb_ext.py` template when you create an external cluster.
```